spec/services/request_password_reset_spec.rb

```diff
@@ -81,6 +81,46 @@
       end
     end
 
+    context 'when the user is found and confirmed, but is suspended' do
+      subject(:perform) do
+        described_class.new(
+          email: email,
+          irs_attempts_api_tracker: irs_attempts_api_tracker,
+        ).perform
+      end
+
+      before do
+        user.suspend!
+        allow(UserMailer).to receive(:reset_password_instructions).
+          and_wrap_original do |impl, user, email, options|
+            token = options.fetch(:token)
+            expect(token).to be_present
+            expect(Devise.token_generator.digest(User, :reset_password_token, token)).
+              to eq(user.reset_password_token)
+
+            impl.call(user, email, **options)
+          end
+      end
+
+      it 'does not send password reset instructions' do
+        expect { subject }.
+          not_to(change { user.reload.reset_password_token })
+      end
```

Comment:
the description here makes it sound like we'd look at an email, I think we should update this to describe the token?
```suggestion
      it 'does not set a new password reset token' do
        expect { subject }.
          not_to(change { user.reload.reset_password_token })
      end
```

Reply:
I agree - I had copied it over from another test and negated it.  I can change it for both tests.